src/__init__.py

```diff
@@ -1,3 +1,18 @@
 class Interpreter(object):
     def parse(self, text):
         raise NotImplementedError()
+
+
+config_keys = [
```

Comment:
Is there a better place for them, `config.py` or something? Wouldn't look for them in this place.